### PR TITLE
update service layer to accommodate the 4 PDF files generated for LAM…

### PR DESCRIPTION
**Pull request by cosletmihai**
… report

### CODE REVIEW

File: lam4doc/config.py [hunk 1]
```diff
@@ -85,11 +86,16 @@ def LAM_HTML_REPORT_TEMPLATE_LOCATION(self) -> str:
...
     @property
     def LAM_PDF_REPORT_TEMPLATE_LOCATION(self) -> str:
-        # correct value will be used when pdf implementation is ready
-        value = str(Path(__file__).parents[1] / 'templates/html')
+        value = str(Path(__file__).parents[1] / 'templates/pdf')
         self.logger.debug(value)
         return value
 
+    @property
+    def LAM_PDF_REPORT_TEMPLATE_LATEX_FILES(self) -> list:
```
Comment (costezki) [inline lam4doc/config.py:94]: I guess this is fairly safe as long as we package the templates together with the code.

File: lam4doc/entrypoints/ui/views.py [hunk 2]
```diff
@@ -51,7 +51,9 @@ def download_lam_report():
         else:
             try:
                 with tempfile.TemporaryDirectory() as temp_folder:
-                    file_name = f'LAM-report.{form.report_extension.data}'
+                    report_extension = form.report_extension.data if form.report_extension.data == HTML_REPORT_TYPE \
+                        else ZIP_REPORT_TYPE
+                    file_name = f'LAM-report.{report_extension}'
```
Comment (costezki) [inline lam4doc/entrypoints/ui/views.py:56]: how many report types we can then have? 3?
HTML, pdf, zip ?

File: lam4doc/services/handlers.py [hunk 3]
```diff
@@ -35,45 +36,71 @@ def get_report_location(extension: str) -> str:
...
     :param report_builder: a report builder type service (eds4jinja2.builders.report_builder.ReportBuilder)
+    :param report_name: optional name for the report
     :return: location of the report document
     """
     logger.debug(f'start service for generating report with location: {location}')
 
     report_builder.make_document()
 
-    report_path = Path(str(location)) / report_builder.template
+    report_path = Path(str(location)) / (report_name if report_name else report_builder.template)
```
Comment (costezki) [inline lam4doc/services/handlers.py:51]: fair enough

**Review by costezki — APPROVED** [review]
looks neat

File: lam4doc/entrypoints/ui/views.py (hunk 2 above)
Comment (cosletmihai) [reply lam4doc/entrypoints/ui/views.py:56]: there are 2 types of formats that the service provides:
- HTML
- ZIP (the zip contains the PDF files)

**Pull request merged**